packages/display/test/DisplayObject.tests.ts

```diff
@@ -266,6 +266,24 @@ describe('DisplayObject', () =>
         );
     });
 
+    describe('nullCheck', () =>
+    {
+        it('should return undefined as transform may be null', () =>
+        {
+            // @ts-expect-error - instantiating DisplayObject
+            const child = new DisplayObject();
+            const container = new Container();
+
+            container.addChild(child);
+
+            container.transform = null;
+            child.transform = null;
```

Comment:
No one should be assigning `null` to those transform props. A better test is probably to destroy the DisplayObject & Container. Would assuming this gives the same result, exercises the same paths.
```suggestion
            container.destroy({ children: true });
```